package.json

```
@@ -74,7 +74,7 @@
     "file-loader": "^6.0.0",
     "fork-ts-checker-webpack-plugin": "^5.0.0 || ^6.0.0",
     "fs-extra": "^9.0.0",
-    "handlebars": "^4.0.11",
+    "handlebars": "4.7.7",
```

```suggestion
    "handlebars": "^4.7.7",
```

For the next versions